src/exojax/utils/exofile.py

```diff
@@ -0,0 +1,54 @@
+"""Exojax File Format
+   
+   Note:
+       EFF contains .json and main file (normally hdf5 for vaex)
+
+   - emif: Exomol MODIT Input Format
+
+"""
+import pathlib
+import vaex
+import json
+
+def save_exomol_mif(output,nu_grid,cnu,indexnu,logsij0,A,alpha_ref,n_Texp,elower,extension=".hdf5"):
+    """Save inputs as MODIT Input File Format for ExoMol (emif)
+
+    Args:
```

Comment:
Can you give Args?

Reply:
I'm afraid that this file is not made by me. 
So I didn't modify it at all. Shall I get it organized?

Reply:
Ah OK, sorry. If you do not need this function, remove it.